bin.js

```diff
@@ -146,6 +147,10 @@ function start (entry, argv, done) {
   var address = argv.address
   var port = argv.port
   var sse = Sse(assets)
+  var certOpts = {
+    installCertutil: true
+  }
+  var certName = entry.replace(new RegExp(path.sep, 'g'), '-').slice(-64)
```

Comment:
So we have to verify that we only ever pass in 64 characters.  If you try to pass in with path separators, it fails as well since the cert name has the directory name attached, and that directory doesn't exist (which might be a bug in devcert).